kinto_emailer/tests/test_includeme.py

```diff
@@ -313,3 +313,110 @@ def test_email_is_when_review_is_requested(self):
+
+
+class HookValidationTest(EmailerTest):
+    def setUp(self):
+        self.valid_collection = {
+            'kinto-emailer': {
+                'hooks': [{
+                    'template': '{user_id} requested review on {uri}.',
+                    'recipients': [
+                        'me@you.com',
+                        'My friend alice <alice@wonderland.com>',
+                        '<t.h.i.s+that@some.crazy.moderndomainnameyouknow>'
+                    ]
+                }]
+            }
+        }
+        self.headers = dict(self.headers, **get_user_headers('nous'))
+        self.app.put('/buckets/b', headers=self.headers)
+
+    def test_supports_simple_recipients_list(self):
+        self.app.put_json('/buckets/b/collections/c',
+                          {'data': self.valid_collection},
+                          headers=self.headers)
+
+    def test_supports_groups_url(self):
+        self.valid_collection['kinto-emailer']['hooks'][0]['recipients'] += [
+            '/buckets/b/groups/g'
+        ]
+        self.app.put_json('/buckets/b/collections/c',
+                          {'data': self.valid_collection},
+                          headers=self.headers)
+
+    def test_supports_empty_list_of_hooks(self):
+        self.valid_collection['kinto-emailer']['hooks'] = []
+        self.app.put_json('/buckets/b/collections/c',
+                          {'data': self.valid_collection},
+                          headers=self.headers)
+
+    def test_supports_empty_template(self):
+        self.valid_collection['kinto-emailer']['hooks'][0]['template'] = ''
+        self.app.put_json('/buckets/b/collections/c',
+                          {'data': self.valid_collection},
+                          headers=self.headers)
+
+    def test_fails_with_missing_hooks(self):
+        self.valid_collection['kinto-emailer'].pop('hooks')
+        r = self.app.put_json('/buckets/b/collections/c',
+                              {'data': self.valid_collection},
+                              headers=self.headers,
+                              status=400)
+        assert 'Missing "hooks"' in r.json['message']
+
+    def test_fails_if_missing_template(self):
+        self.valid_collection['kinto-emailer']['hooks'][0].pop('template')
+        r = self.app.put_json('/buckets/b/collections/c',
+                              {'data': self.valid_collection},
+                              headers=self.headers,
+                              status=400)
+        assert 'Missing "template"' in r.json['message']
+
+    def test_fails_with_empty_list_of_recipients(self):
+        self.valid_collection['kinto-emailer']['hooks'][0]['recipients'] = []
+        r = self.app.put_json('/buckets/b/collections/c',
+                              {'data': self.valid_collection},
+                              headers=self.headers,
+                              status=400)
+        assert 'Empty list of recipients' in r.json['message']
```

Comment:
Can we use the error validator there? So that we can make sure about all the JSON response?